fast64_internal/f3d/f3d_gbi.py

```diff
@@ -4236,7 +4236,10 @@ def geoFlagListToWord(flagList, f3d):
         if name in f3d.allGeomModeFlags:
             word += getattr(f3d, name)
         else:
-            raise PluginError("Invalid geometry mode flag " + name)
+            try:  # Try to cast name to an int instead, if this fails raise an explicit error
+                word += int(word, 0)
+            except:
+                raise PluginError("Invalid geometry mode flag " + name)
```

Comment:
```suggestion
            except ValueError as e:
                raise PluginError("Invalid geometry mode flag " + name) from e
```
Catch-all except is bad practice, and chaining exceptions like this gives more helpful context in the trace

Reply:
Fair, in this case the only exception that needs to be catched is value error, my bad